From 89c95af4d7adae1e0a8eddb5f303caca295664a8 Mon Sep 17 00:00:00 2001
From: Wanchang Ryu <wanchang.ryu@lge.com>
Date: Tue, 24 Nov 2020 14:18:06 +0900
Subject: [PATCH] [op][ds][build] Added mksnapshot.gn for mksnapshot target

:Release Notes:
Added mksnapshot.gn for mksnapshot target

:Detailed Notes:
Chromium default BUILD.gn contains many dependency to required component
for building chromium but theses are not required to build native
mksnapshot binary.
mksnapshot.gn is used as gn dotfile for building mksnapshot target.

:Testing Performed:
Build webos images for various targets

:QA Notes:
N/A

:Issues Addressed:
[PLAT-78282] mksnapshot from webruntime fails to run when host and
target have different glibc version
---
 src/mksnapshot.gn | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 src/mksnapshot.gn

diff --git a/src/mksnapshot.gn b/src/mksnapshot.gn
new file mode 100644
index 0000000000..c63bf6d4de
--- /dev/null
+++ b/src/mksnapshot.gn
@@ -0,0 +1,9 @@
+declare_args() {
+  # The v8 snapshot needs to be built by code that is compiled with a
+  # toolchain that matches the bit-width of the target CPU, but runs on
+  # the host.
+  v8_snapshot_toolchain = ""
+}
+
+buildconfig = "//build/config/BUILDCONFIG.gn"
+root = "//v8"
-- 
2.17.1

